packages/lib/src/__tests__/api-helpers.test.ts

```diff
@@ -0,0 +1,235 @@
+// import { mocked } from 'ts-jest/utils'
```

Comment:
Could remove this, otherwise looks good.

Reply:
fixed